tmt/steps/provision/minute.py

```diff
@@ -111,39 +157,52 @@ def show(self):
 
     def wake(self, data=None):
         """ Override options and wake up the guest """
-        super().wake(['image', 'flavor', 'allow_ipv4_only'])
+        super().wake(['image', 'flavor', 'allow_ipv4_only', 'list', 'list-flavors'])
         if self.opt('dry'):
             return
 
         # Read API URL from 1minutetip script
-        try:
-            self.debug(f"Get the API URL from '{SCRIPT_PATH}'.")
-            script_content = self.read(SCRIPT_PATH)
-            match = re.search(API_URL_RE, script_content)
-            if not match:
-                raise tmt.utils.ProvisionError(
-                        f"Could not obtain API URL from '{SCRIPT_PATH}'.")
-            self.data['api_url'] = match.group('url')
-            self.debug('api_url', self.data['api_url'], level=3)
-        except tmt.utils.FileError:
-            raise tmt.utils.ProvisionError(
-                f"File '{SCRIPT_PATH}' not found. Please install 1minutetip.")
+        self._populate_api_url()
 
         if data:
+            data['api_url'] = self.api_url
             self._guest = GuestMinute(data, name=self.name, parent=self.step)
             self._guest.wake()
 
     def go(self):
-        """ Provision the container """
+        """ Provision the container, list flavors or list of images probided by openstack """
         super().go()
 
+        self._populate_api_url()
+
+        for opt in ['list', 'list-flavors']:
+            val = self.get(opt)
+            if opt == 'list-flavors' and val:
```

Comment:
Maybe message about what is being printed as a header?

Reply:
How do you think that header should look? Maybe people know and remember what arguments they used? So they can save some time when they just don't want to get rid of header every time in script or something?

Reply:
I'm in tmt repo and I do `tmt run provision -h minute --list`. Output is
```
/var/tmp/tmt/run-057

/plans/features/advanced
    provision
        how: minute
1MT-Fedora-28
```

So the header is already there. Maybe output like this instead?
```
/var/tmp/tmt/run-057

/plans/features/advanced
    provision
        how: minute
        Available images:
            1MT-Fedora-28
```

Reply:
Eh, I was replying with images to flavors. But the idea is same for both.

Reply:
Yes... some kind of "header" is here but dunno how to remove it...
So what do you suggest? Because I don't know which header I should add.